gemrb/plugins/2DAImporter/2DAImporter.cpp

```diff
@@ -102,7 +102,9 @@ bool p2DAImporter::Open(DataStream* str)
 			row++;
 		}
 	}
+
 	delete str;
+	assert(rows.size() < std::numeric_limits<index_t>::max());
```

Comment:
I don't think we need to be this paranoid. You'd need specially crafted data and tons of free memory to be able to fail here.

Reply:
Force of habit when it comes to _all input is evil_ :shrug: